Oskar gdev 2548 risk scoring tool

oskar700 opened the pull request:
# Pull request questions

## Which issue does this address 

Issue number: GDEV-2548

## Why was change needed

This is to service customer need.

## What does change improve

Adds ability to monitor the quality of data.

barrycaceres commented on src/main/java/com/senzing/calculator/scoring/risk/service/RiskScoringService.java:
```diff
@@ -0,0 +1,370 @@
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
```
as an FYI, Java does have a standard JSON parsing api (javax.json).  Don't know if using org.json is a requirement or not -- for the API server we use org.glassfish implementation of javax.json.

oskar700 replied:
I replaced the parser.

oskar700 commented on .project:
```diff
@@ -1,4 +1,23 @@
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
+	</natures>
 </projectDescription>
```
I added .project into .gitignore.   Also backed the .project change out.

oskar700 commented on pom.xml:
```diff
@@ -0,0 +1,132 @@
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20090211</version>
```
I replaced the parser and use not the one you recommend.

oskar700 commented:
Thanks for reviewing Barry.  Regarding the destroy() call, there is only one instance that lives for the lifetime of the app, but I added this and call on exit.  It will be good if more threads, or functionality changes in the future.
I replaced the json.org parser with java.json.
